docs/components/radio.md

```diff
@@ -201,18 +194,23 @@ Any valid attribute will be added to the input. You can read more about `$attrs`
   Label goes here. The radio is {{ checked ? "selected" : "not selected" }}
 </KRadio>
 ```
+:::warning NOTE
+Avoid slotting in block-level elements like `div` into a `default` slot as it will be rendered inside the `label` element. This applies to card-style radio as well.
```

Comment:
nit: wording
```suggestion
To preserve a valid HTML structure, avoid slotting in block-level elements such as a `div` into the `default` slot as it will be rendered inside a `label` element. This also applies to card-style radio.
```

Reply:
Done